go/stellar/global.go

```diff
@@ -108,6 +109,11 @@ func (s *Stellar) OnDbNuke(mctx libkb.MetaContext) error {
 	return nil
 }
 
+func (s *Stellar) Shutdown(mctx libkb.MetaContext) error {
+	s.Clear(mctx)
+	return nil
+}
```

Comment:
adding this was necessary to stop the autoclaimrunner, which logged after a test completed.